src/Expression.php

```diff
@@ -41,7 +41,7 @@ class Expression implements \ArrayAccess, \IteratorAggregate
      *
      * @var string
      */
-    protected $escape_char = '"';
+    protected $quotedIdentifier = '"';
```

Comment:
Identifier? `escapeChar` seems better to me, putting here something more than a char probably is wrong.

Reply:
This is not an escape string but quoted identifier.
https://dba.stackexchange.com/questions/23129/benefits-of-using-backtick-in-mysql-queries

Reply:
Is this guarantted to be never used with values and only if names?

Reply:
This string is used to wrap (quote) literal strings to be used in the query like in MySQL
```
CREATE TABLE `my table` (id INT);
```

Reply:
If, then `quoteIdentifier`.

But Microsoft https://docs.microsoft.com/en-us/sql/t-sql/statements/set-quoted-identifier-transact-sql uses this term for string values not string identifiers..